File: 2058-concatenation-of-array/2058-concatenation-of-array.py
class Solution(object):
    def getConcatenation(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        ans=[]
        i=0
        while i<2:
            for j in range(len(nums)):
                ans.append(nums[j])
            i=i+1
        return ans

File: 2058-concatenation-of-array/2058-concatenation-of-array.java
import java.util.*;
class Solution {
    public int[] getConcatenation(int[] nums) {
     int i,j,c=0;
     int[] temp = new int[2*(nums.length)];
     for(i=0;i<2;i++)
     {
        for(j=0;j<nums.length;j++)
        {
            temp[c]=nums[j];
            c=c+1;
        }
     }
     return temp;
    }
}